doc-content/jbpm-docs/src/main/asciidoc/BAM/Importing-dashbuilder-data-proc.adoc

```diff
@@ -38,7 +38,8 @@ endif::[]
 
 ifdef::DM[]
 
-= Importing Dashbuilder data
+= Importing {CENTRAL} dashboards
+//We are really importing dashboards, right? To call it dashboard data I think is confusing with the data set section.
```

Comment:
I'd say that dashboards and dashboard data could be interchangeable but we should use just one of these terms in all places

Reply:
Updated, please review.